src/types/classesObject.js

```diff
@@ -0,0 +1 @@
+export type classesObject = {[string]: string}
```

Comment:
Types should always start with an uppercase character. Also, I think it's better to name it `Classes` as it's pretty clear that's an object.

Do we really need an extra file for this one type?

Reply:
@HenriBeck thanks.
Should I put it inside the `/types/jss.js` file?

Reply:
yes.